src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+const { Token } = require('../models/token.js');
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+
+    return;
```

Comment:
Consider returning a value or a confirmation message from the `save` function to indicate that the token has been successfully created or updated. This can be useful for debugging or logging purposes.